client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -62,17 +64,18 @@ export default function Confirm( props: WooCommerceInstallProps ): ReactElement
 		wpcomSubdomainWarning,
 		siteUpgrading,
 		hasBlockers,
-
+		isDataReady,
 		warnings,
-		isReadyForTransfer,
+		isAtomicSite,
 	} = useWooCommerceOnPlansEligibility( siteId );
 
-	// Skip to transfer step if the site is ready for transfer.
 	useEffect( () => {
-		if ( isReadyForTransfer ) {
-			goToStep( 'transfer' );
+		if ( ! isAtomicSite ) {
+			return;
```

Comment:
It's not allowing me to make a suggestion for all of it, because of the way the diff has worked out, but shall we flip the logic again?

```suggestion
		if ( isAtomicSite ) {
			page.redirect( `/woocommerce-installation/${ wpcomDomain }` );
```

Reply:
Honestly, I like this order. I prefer to keep it as it is unless you really think we do really should change it :-D